jetbrains-rider/ReSharper.AWS/src/AWS.Daemon/RunMarkers/LambdaRunMarkerProvider.cs

```diff
@@ -0,0 +1,371 @@
+using System.Collections.Generic;
+using System.Linq;
+using JetBrains.Annotations;
+using JetBrains.Application.Settings;
+using JetBrains.Metadata.Reader.API;
+using JetBrains.Metadata.Reader.Impl;
+using JetBrains.ProjectModel;
+using JetBrains.ProjectModel.ProjectsHost.Dependencies;
+using JetBrains.ReSharper.Feature.Services.Daemon;
+using JetBrains.ReSharper.Host.Features.RunMarkers;
+using JetBrains.ReSharper.Psi;
+using JetBrains.ReSharper.Psi.Caches.SymbolCache;
+using JetBrains.ReSharper.Psi.CSharp;
+using JetBrains.ReSharper.Psi.CSharp.Tree;
+using JetBrains.ReSharper.Psi.Resolve;
+using JetBrains.ReSharper.Psi.Tree;
+using JetBrains.ReSharper.Psi.Util;
+using JetBrains.Util;
+using JetBrains.Util.Logging;
+using IMethodDeclaration = JetBrains.ReSharper.Psi.CSharp.Tree.IMethodDeclaration;
+
+namespace ReSharper.AWS.RunMarkers
+{
+    [Language(typeof(CSharpLanguage))]
```

Comment:
We technically have PowerShell as a Lambda runtime, if we added that would it go in resharper too?

Reply:
@abrooksv, Reshaper don't have PowerShell support. There is an external plugin that add frontend (Idea) support in Rider - https://plugins.jetbrains.com/plugin/10249-powershell.

Reply:
Okay.